aicsimageio/readers/mrc_reader.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Any, Tuple
+
+import mrcfile
+import dask.array as da
+import xarray as xr
+from numpy.lib.recfunctions import structured_to_unstructured
+from fsspec.spec import AbstractFileSystem
+from fsspec.implementations.local import LocalFileSystem
+
+from .. import exceptions, types
+from ..utils import io_utils
+from .reader import Reader
+
+
+class MrcReader(Reader):
+    """
+    Read MRC files using mrcfile.
+
+    Parameters
+    ----------
+    image: types.PathLike
+        Path to image file to construct Reader for.
+    """
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            with mrcfile.open(path, permissive=True, header_only=True):
+                return True
+        except (ValueError, TypeError):
+            return False
+
+    def __init__(self, image: types.PathLike):
+        # Expand details of provided image
+        self._fs, self._path = io_utils.pathlike_to_fs(image, enforce_exists=True)
+        # Catch non-local file system
+        if not isinstance(self._fs, LocalFileSystem):
+            raise NotImplementedError(
+                f"mrc reader not yet implemented for non-local file system. "
+                f"Received URI: {self._path}, which points to {type(self._fs)}."
+            )
+
+        if not self._is_supported_image(self._fs, self._path):
+            raise exceptions.UnsupportedFileFormatError(
+                self.__class__.__name__, self._path
+            )
+
+    @property
+    def scenes(self) -> Tuple[str, ...]:
+        # TODO: treat mrcs as stacks of scenes?
```

Comment:
I don't know much about the MRCS format. Can you explain the difference between MRC and MRCS?

Reply:
It's exactly the same format. The only difference is that (as far as I know, by convention, mainly from the RELION software), an `s` at the end stands for "stack", meaning that subsequent `z` slices are actually spatially unrelated 2D images.

Reply:
Wonderful. I would say implement them as scenes if possible then!

Reply:
Implemented. Turns out, the format was more complex that I thought :) but now it properly handles single images, image stacks, single volumes and stacks of volumes.